content/en/docs/faas/lambda-collector.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Lambda Collector Configuration
+linkTitle: Lambda Collector Configuration
```

Comment:
```suggestion
linkTitle: Lambda Collector Config
```

Reply:
Would it be ok to shorten the `linkTitle` of the three newly added leaf pages by replacing "Lambda" by λ?